forms/tests/test_api.py

```diff
@@ -174,3 +174,14 @@ def test_answer_post(admin_client, admin_user, basic_form):
     url = reverse("answer-list")
     response = admin_client.get(url)
     assert response.status_code == 200
+
+    user = user_factory()
+    password = "test"
+    user.set_password(password)
+    user.save()
+    admin_client.logout()
```

Comment:
There is also just a "client". No admin client is needed to use here.

https://pytest-django.readthedocs.io/en/latest/helpers.html#client-django-test-client